suse_openstack_cloud

```diff
@@ -396,13 +396,12 @@ fi
 section_header "RabbitMQ broker"
 
 if [ -e /usr/sbin/rabbitmqctl ]; then
```

Comment:
since you're not runnign that command but ```/usr/lib64/rabbitmq/bin/rabbitmqctl```, it would be good to check for that one instead